src/AddTodoForm.js

```diff
@@ -1,11 +1,22 @@
 import React from "react";
 
-const AddToDoForm = () => {
+const AddToDoForm = (props) => {
+
+	const handleAddToDo = (event) => {
+		event.preventDefault();
+		let todoTitle = event.target.title.value;
+		console.log(todoTitle);
```

Comment:
you left a logging statement, maybe you want to use eslint to avoid that (it's a linter)

Reply:
Thanks for the suggestion. Do you have any thoughts on which style guide I should use?